src/helper.js

```diff
@@ -0,0 +1,82 @@
+
+const findParent = (item, selectedOption, category, tree, path, callback) => {
+  for (let i = tree.length - 1; i >= 0; i--) {
+    (function () {
+      var currentPath = path.slice();
```

Comment:
What is the use of this `slice`

Reply:
it creates a copy of path in each loop, if slice is not given it overrides with all the previous values along with new values.